docs/docs/gatsby-image-api.md

```diff
@@ -0,0 +1,344 @@
+
+Once you've queried for an image to retrieve its data, you can pass the `fluid` data into the `Img` component:
+
+```jsx
+import { useStaticQuery } from "gatsby"
+import Img from "gatsby-image"
+
+export default () => {
+  const data = useStaticQuery(query`
+    file(relativePath: { eq: "images/default.jpg" }) {
+      childImageSharp {
+        # Specify a fluid image and fragment
+        // highlight-start
+        fluid {
+          ...GatsbyImageSharpFluid
+        }
+        // highlight-end
+      }
+    }
+  `)
+  return (
+    <div>
+      <h1>Hello gatsby-image</h1>
+      <Img
+        fluid={data.file.childImageSharp.fluid} {/* highlight-line */}
+        alt="Gatsby Docs are awesome"
+      />
+    </div>
+  )
+}
+```
+
+Read more in the [gatsby-plugin-sharp](/packages/gatsby-plugin-sharp/?=#fluid) README.
+
+#### Fluid image query parameters
+
+In a query, you can specify options for fluid images.
+
+- `maxWidth` (int, default: 800)
+- `maxHeight`(int)
+- `quality` (int, default: 50)
+- `srcSetBreakpoints` (array of int, default: [])
+- `fit` (string, default: `[sharp.fit.cover][6]`)
+- `background` (string, default: `rgba(0,0,0,1)`)
+
+#### Returns
+
+- `src` (string)
+- `width` (int)
+- `height` (int)
+- `aspectRatio` (float)
+
+This is where fragments like `GatsbyImageSharpFluid` come in handy, as they'll return all the above items in one line:
+
+```graphql
+file(relativePath: { eq: "images/default.jpg" }) {
+  childImageSharp {
+    // highlight-next-line
+    fluid(maxWidth: 400) {
+      // highlight-next-line
+      ...GatsbyImageSharpFluid
+    }
+  }
+}
+```
+
+### Resized images
+
+In addition to _fixed_ and _fluid_ images, the gatsby-image API allows you to call a `resize` method with `gatsby-plugin-sharp` to return a single image as opposed to multiple sizes. There are no default fragments available for the resize method.
+
+#### Parameters
+
+- `width` (int, default: 400)
+- `height` (int)
+- `quality` (int, default: 50)
+- `jpegProgressive` (bool, default: true)
+- `pngCompressionLevel` (int, default: 9)
+- `base64`(bool, default: false)
+
+#### Returns
+
+Resize returns an object with the following items:
+
+- `src` (string)
+- `width` (int)
+- `height` (int)
+- `aspectRatio` (float)
+
+```graphql
+allImageSharp {
+  edges {
+    node {
+        resize(width: 150, height: 150, grayscale: true) {
+          src
+        }
+    }
+  }
+}
+```
+
+### Shared query parameters
+
+In addition to `gatsby-plugin-sharp` settings in `gatsby-config.js`, there are additional query options that apply to both _fluid_ and _fixed_ images:
+
+- `grayscale` (bool, default: false)
+- `duotone` (bool|obj, default: false)
```

Comment:
This might be too much, but a before/after of `grayscale` and `duotone` would be dope.

Reply:
ooh maybe @gillkyle could help me with that!

Reply:
Here's what I added (note: these images will only be displayed like this if you've upgraded `gatsby-remark-images` like this PR #14716):
![example-images-screenshot](https://user-images.githubusercontent.com/21114044/59311683-a8060e00-8c67-11e9-9aee-5ccdf970cb57.png)

Just pushed right to this branch, feel free to edit it if you want.

Reply:
you're amazing, thanks @gillkyle !